build.gradle

```diff
@@ -22,9 +22,14 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.synopsys.integration:jenkins-common:0.5.4'
+    implementation 'com.synopsys.integration:jenkins-common:0.5.5'
     implementation 'org.apache.commons:commons-text:1.10.0'
-    implementation 'com.vladsch.flexmark:flexmark-all:0.64.0'
+
+    implementation('com.vladsch.flexmark:flexmark-all:0.64.0') {
+        exclude group: 'com.ibm.icu', module: 'icu4j'
+    }
+
```

Comment:
I would have thought that line 32 would have sufficed (without line 29), but you may have already proven otherwise.

Reply:
It's actually a REALLY good question because the way it's shown above... seems to be the "most popular" solution in my google searches...